indexer/services/l1/confirmed_headers.go

```diff
@@ -17,9 +17,8 @@ import (
 )
 
 const (
-	DefaultConnectionTimeout        = 30 * time.Second
-	DefaultConfDepth         uint64 = 20
-	DefaultMaxBatchSize             = 100
+	DefaultConnectionTimeout = 30 * time.Second
```

Comment:
Should these be defined in `op-service`?

Reply:
I think these are specific to the indexer itself, so should remain here - they refer to calls only the indexer makes.